app/src/main/java/org/jellyfin/androidtv/ui/LiveProgramDetailPopup.java

```diff
@@ -337,6 +344,37 @@ public void onClick(View v) {
         return tune;
     }
 
+    public android.widget.ImageButton createFavoriteButton() {
```

Comment:
Why not just "ImageButton"?

Reply:
I looked into this more and initially it was because I was lazy and didn't look into the `org.jellyfin.androidtv.ui.ImageButton` class. After experimenting with that class the default android ImageButton is what should be used here. The other buttons('Tune to Channel', 'Record', etc) are default android Buttons so the styling matches. In this file ImageButton defaults to the jellyfin image button so the full package name has to be specified for the android version.

So if you are asking why I got specific with the package it's becuase `ImageButton` maps to the jellyfin specific class. If you are asking why I don't use the jellyfin ImageButton, it does not match the style of the rest of the popup.

Reply:
I didn't realize we had our own ImageButton in the same package. It makes sense now to me!